.jshintrc

```diff
@@ -1,6 +1,11 @@
 {
+	"globals": {
+		"Image"		: true,
```

Comment:
Globals should be set on a per-file basis in the header area, these are not universally known globals throughout the project.